### PR TITLE
note_input_optimization_2

### PR DESCRIPTION
**Pull request by RomanPudashkin**
Resolves: https://github.com/musescore/MuseScore/issues/12841

### CODE REVIEW

**Review by vpereverzev — APPROVED** [review]

**Comment by DmitryArefiev** [comment]
Tested #12841 + additional tests with playback, MIDI input, MuseSounds on Win10, Mac14, **LinuxUbuntu 22.04**

CPU usage **when idle** on Linux is **x2 less than in the master** (European Concert Band template with MS Basic)

File: src/framework/audio/audiotypes.h [hunk 1]
```diff
@@ -58,6 +58,8 @@ using aux_channel_idx_t = uint8_t;
 using PlaybackData = std::variant<mpe::PlaybackData, io::IODevice*>;
 using PlaybackSetupData = mpe::PlaybackSetupData;
 
+static constexpr TrackId INVALID_TRACK_ID = -1;
```
Comment (Jojo-Schmitz) [inline src/framework/audio/audiotypes.h:61]: I know, too late once again, but why not `mu::nidx`?

Comment (Jojo-Schmitz) [reply src/framework/audio/audiotypes.h:61]: Here or even instead of `INVALID_TRACK_ID`

**Comment by benjibasson83** [comment]
Hi, the bug #12841 still occure from Musescore 4.0 untile the last musescore 4.4.4. 
Somebody say that  note_input_optimization_2 #19740  resolve issue #12841 althought the last 4.4.4 musescore consume 25% of CPU when idle.
Can someone please make a tuto to solve the issue #12841
How to CPU usage when idle on Linux is x2 less than in the master (European Concert Band template with MS Basic) with stable version instead of private nightly , without modify source code audiotypes.h and without compil?

**Pull request merged**